jetty-core/jetty-rewrite/src/main/java/org/eclipse/jetty/rewrite/handler/RedirectRegexRule.java

```diff
@@ -63,6 +65,27 @@ public void setLocation(String location)
         _location = location;
     }
 
+    /**
+     * <p>Is the input URI query merged with replacement URI query</p>
+     *
+     * @return true to merge input query with replacement query.
+     */
+    public boolean isAddQueries()
```

Comment:
isMergeQueries might be a better name

Reply:
doesn't "merge queries" reorg the entire query line if keys of the same name are present on both sides?

In other words ...

* line 1 is `type=bar?mode=foo`
* line 2 is `type=zed?quantity=22`

With merge queries you can wind up with `type=bar&type=zed&mode=foo&quantity=22`  (what servlet does)
With add queries you get `<line1><line2>` so `type=bar?mode=foo&type=zed?quantity=22` (what rewrite does)

Reply:
Then you need to change the javadoc and use the expression "add" instead of "merge"  !